app-modules/archeo/src/Listeners/CompressPdfOnUploadListener.php

```
@@ -0,0 +1,30 @@
+        if (
+            $media->model_type !== Activity::class
+            || $media->collection_name !== 'pdfs'
+            || $media->mime_type !== 'application/pdf'
+        ) {
+            return;
+        }
+
+        /** @var User|null $user */
+        $user = Auth::user();
+
+        CompressPdfJob::dispatch($media->id, $user);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Metafori\Archeo\Jobs\CompressPdfJob' in method 'handle'. (undefined)

(StaticAccess)